Felienne/hedy
# [BUG FIX] Parsons attempts are correctly stored in database

**Description**
Currently Parsons attempts in the database are overwritten for each user on each attempt. This is due to the `username` field being the partition_key. We fix this by adding a random 12 char string as id to each entry, setting this as the partition key. We then add `username` as the index key to still enable filtering on username for analysis later on. Lastly we add the `correct` field being either 0 or 1 indicating if the answer was correct.

**Fixes**
This PR fixes #2832.

**How to test**
Deploy to Alpha, verify that new attempts are no longer overwriting the old ones (in the Dynamo database).
